cmd/util/ledger/migrations/change_contract_code_migration.go

```diff
@@ -292,64 +324,64 @@ func SystemContractChanges(chainID flow.ChainID, options SystemContractChangesOp
 			Address:      lockedTokensAddress,
 			ContractName: "LockedTokens",
 			NewContractCode: string(coreContracts.FlowLockedTokens(
-				systemContracts.FungibleToken.Address.HexWithPrefix(),
-				systemContracts.FlowToken.Address.HexWithPrefix(),
-				systemContracts.IDTableStaking.Address.HexWithPrefix(),
+				fungibleTokenAddress.HexWithPrefix(),
+				flowTokenAddress.HexWithPrefix(),
+				idTableStakingAddress.HexWithPrefix(),
 				stakingProxyAddress.HexWithPrefix(),
-				systemContracts.FlowStorageFees.Address.HexWithPrefix(),
+				flowStorageFeesAddress.HexWithPrefix(),
 			)),
 		},
 
 		// token related contracts
 		NewSystemContractChange(
 			systemContracts.FlowFees,
 			coreContracts.FlowFees(
-				systemContracts.FungibleToken.Address.HexWithPrefix(),
-				systemContracts.FlowToken.Address.HexWithPrefix(),
-				systemContracts.FlowStorageFees.Address.HexWithPrefix(),
+				fungibleTokenAddress.HexWithPrefix(),
+				flowTokenAddress.HexWithPrefix(),
+				flowStorageFeesAddress.HexWithPrefix(),
 			),
 		),
 		NewSystemContractChange(
 			systemContracts.FlowToken,
 			coreContracts.FlowToken(
-				systemContracts.FungibleToken.Address.HexWithPrefix(),
+				fungibleTokenAddress.HexWithPrefix(),
 				fungibleTokenMetadataViewsAddress.HexWithPrefix(),
-				systemContracts.MetadataViews.Address.HexWithPrefix(),
-				systemContracts.ViewResolver.Address.HexWithPrefix(),
+				metadataViewsAddress.HexWithPrefix(),
+				burnerAddress.HexWithPrefix(),
```

Comment:
flow-ft's contract code "constructor function" changed in a recent commit: the argument is no longer the address of the view resolver, but the one of the burner